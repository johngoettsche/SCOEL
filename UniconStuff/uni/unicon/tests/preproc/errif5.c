/* test error message: #endif with no #if */

xxx
#endif
