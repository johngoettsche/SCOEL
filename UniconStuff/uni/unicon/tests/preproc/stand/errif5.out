#line 2 "errif5.c"

xxx
pp: File errif5.c; Line 4: invalid context for #endif
